packages/typeahead/src/FzTypeahead.vue

```diff
@@ -99,20 +161,23 @@ const debounceHandleInput = debounce(
   props.delayTime,
 );
 
-function handleInput(val: string) {
+function handleInput(val: string, isOpen: boolean) {
   inputValue.value = val;
-  const selected = internalOptions.value.find((opt) => opt.value === val);
+  const selected = internalOptions.value?.find((opt) => opt.value === val);
   if (!selected) model.value = undefined;
   debounceHandleInput(val);
+  if (!isOpen) {
```

Comment:
are you sure you can use `isOpen` here? I think it is not possible, because it is not accessible outside the scope of FzSelect's slot. If you want two-way communication, you should probably use a v-model or provide/inject

Reply:
sure you can, I pass isOpen to the function called inside the named slot. I don't want two way communication: i am just reading its state and calling FzSelect's `forceOpen` method if needed be

Reply:
I thought it was "global", i missed the function call, my bad